homeassistant/components/edl21/sensor.py

```diff
@@ -384,12 +421,23 @@ async def add_entities(self, new_entities: list[EDL21Entity]) -> None:
 class EDL21Entity(SensorEntity):
     """Entity reading values from EDL21 telegram."""
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self.device_name)},
+            name=self.device_name,
+        )
+
```

Comment:
Please keep this for a follow-up PR

Reply:
By this you mean I should remove those changes from this PR and instead create a new PR for those changes?

Reply:
Removed as suggested, will create a new PR for those changes.